Traceback (most recent call last):
  File "/home/zimmer/neutrino_clustering_V2/number_density_sphere.py", line 228, in <module>
    dPsi_batches = [
  File "/home/zimmer/neutrino_clustering_V2/number_density_sphere.py", line 229, in <listcomp>
    np.load(f'{TEMP_DIR}/dPsi_grid_batch{b}.npy') for b in idx_chunks
  File "/home/zimmer/.conda/envs/neutrino_clustering/lib/python3.9/site-packages/numpy/lib/npyio.py", line 407, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '/gpfs/home4/zimmer/neutrino_clustering_V2/L012N376/DMONLY/CDM_TF50/temp_data_4TZ1/dPsi_grid_batch0.npy'
slurmstepd: error: Detected 1 oom-kill event(s) in StepId=1799362.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
